tests/test_routing.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import time
+import threading
+import sys
+import web
+import urllib
```

Comment:
OPTIONAL: run [isort](http://isort.readthedocs.io) on the imports.

Reply:
Seems `isort` is not Black compatible.

```
$ black --diff db.py
--- db.py	2019-09-22 02:42:22.727265 +0000
+++ db.py	2019-09-22 02:42:36.098191 +0000
@@ -8,12 +8,11 @@
 import os
 import re
 import time
 
 from .py3helpers import PY2, iteritems, numeric_types, string_types, text_type
-from .utils import (iterbetter, iters, safestr, safeunicode, storage,
-                    threadeddict)
+from .utils import iterbetter, iters, safestr, safeunicode, storage, threadeddict
```

`isort` generates the `-` lines, but Black generates `+` lines.

Reply:
__isort --line_length 88__

Reply:
Cool, didn't realize the `--line_length` argument.
Could you like to help review and merge this? This makes web.py familiar with Flask.